internal/deviations/deviations.go

```diff
@@ -196,6 +196,8 @@ var (
 
 	BGPMD5RequiresReset = flag.Bool("deviation_bgp_md5_requires_reset", false, "Device requires a BGP session reset to utilize a new MD5 key")
 
+	QOSDroppedOctets = flag.Bool("deviation_qos_dropped_octets", false, "QOS Dropped octets stats for interface")
```

Comment:
What does this mean? Can you help add more context?

Reply:
The leaf 'dropped octets' is a new leaf added recently for these tests, this is not supported on some platforms and so the check for the leaf is is skipped if this is set to true.